crypto/crypto_aesctr.c

```diff
@@ -95,7 +101,11 @@ crypto_aesctr_stream_cipherblock_generate(struct crypto_aesctr * stream)
 {
 
 	/* Prepare counter. */
-	be64enc(stream->pblk + 8, stream->bytectr / 16);
+	stream->pblk[15]++;
```

Comment:
I did try putting this inside an `if ... else` construct.  For some buffer sizes, it had no effect; for others, it degraded performance slightly.  I find the current construct slightly easier to understand, but I have no qualms about switching to the other one if you prefer that.